link/src/main/java/com/stripe/android/link/injection/NamedConstants.kt

```diff
@@ -24,7 +24,7 @@ const val CUSTOMER_PHONE = "customerPhone"
  * Identifies the shipping address passed in from the customer, used to pre-fill address forms.
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-const val SHIPPING_VALUES = "shippingValues"
+const val SHIPPING_VALUES = "linkShippingValues"
```

Comment:
This is added to the graph but not injected anywhere anymore. Can it be deleted?